frontend/src/views/WheresMyGeneV2/components/Main/index.tsx

```diff
@@ -354,6 +364,8 @@ export default function WheresMyGene(): JSX.Element {
               setEchartsRendererMode={setEchartsRendererMode}
               allChartProps={allChartProps}
               availableFilters={availableFilters}
+              tissues={Object.keys(tissuesByName)}
```

Comment:
`Object.keys(tissuesByName)` creates a new array every render, which will cause unnecessary re-rendering, maybe worth `useMemo` it!